coreos-base/oem-gce/files/files/sudoers-file

```diff
@@ -0,0 +1 @@
+#includedir /var/google-sudoers.d
```

Comment:
The FHS wants this in `/var/lib`.  How hard is it to change?

Reply:
wouldn't be too bad, I'll add it.